websites/H/Hulu/dist/metadata.json

```diff
@@ -1,9 +1,14 @@
 {
+  "$schema": "https://schemas.premid.app/metadata/1.0",
   "author": {
-    "name": "Alanexei ✔",
-    "id": "163319338403627008"
+    "name": "GURRMAN375D",
+    "id": "446528923535278082"
   },
   "contributors": [
+    {
+      "name": "Alanexei ✓",
+      "id": "163319338403627008"
+    },
```

Comment:
Did alan agree to this?

Reply:
https://github.com/PreMiD/Presences/pull/1725